ethereumj-core/src/main/java/org/ethereum/net/rlpx/Node.java

```diff
@@ -103,6 +105,14 @@ public Node(byte[] rlp) {
         this.id = idB;
     }
 
+    /**
+     * @return true if this node is endpoint for discovery loaded from config
+     */
+    public boolean isDiscoveryNode() {
+        // discovery node has nodeId with doubled size
+        return id.length > 64;
+    }
```

Comment:
Honestly I don't like the idea of making ID 128 bytes length. All code (including user) relies on 64 bytes and can be broken at any unpredictable place. 
Can we identify Node by its host(IP)+port+nodeId (i.e. at least change the `equals` method) and set the fake `0x00 x 64` ID? Or at least set the first 32 bytes to zero for fake ID? I'd prefer the first